### ISSUE TITLE
Transition of velodyne release repository

### ISSUE THREAD

**Issue by wep21** [ros2-gbp/velodyne-release#1]
The velodyne release repository is under [ros-drivers-gbp]( https://github.com/ros-drivers-gbp/velodyne-release), not under [ros2-gbp](https://github.com/ros2-gbp/velodyne-release).

**Comment by wep21** [comment]
@JWhitleyWork @clalancette @nuclearsandwich Can I move it to [ros2-gbp](https://github.com/ros2-gbp/velodyne-release)?

**Comment by nuclearsandwich** [comment]
Since the repository here is completely empty with the exception of an automatically created initial commit, I'll plan to `--mirror` push to this repository (destroying that trivial commit) from https://github.com/ros-drivers-gbp/velodyne-release

**Comment by JWhitleyWork** [comment]
Sounds good to me.

**Comment by JWhitleyWork** [comment]
Let me know when this is done and I will modify the rosdistro entries to point to this repo.

**Comment by wep21** [comment]
@nuclearsandwich friendly ping

**Comment by nuclearsandwich** [comment]
@wep21 thanks for the ping, it was needed here. This push has been done.

**Comment by wep21** [comment]
@nuclearsandwich Thank you! Is blooming to `rolling` again necessary to create branches for `jammy`?
@JWhitleyWork Could you modify the target repos in rosdistro?

**Comment by JWhitleyWork** [comment]
@wep21 ros/rosdistro#32699 basically resolved this, right?

**Comment by wep21** [comment]
@JWhitleyWork Yes, but it's not been released on `rolling` yet. Do you have any resource to make release?
